src/import-data/protected-data-utils.ts

```diff
@@ -5,6 +5,8 @@ import { ImportRequest } from './import-types';
 
 type ProtectedSource = { type: 'http'; host: string } | { type: 's3'; bucket: string };
 
+export type importSource = 'anvil' | '';
```

Comment:
nit 😁 capitalize type declaration?

Reply:
as;ldkfj;saldkfj
im actually surprised the linter didn't pick that up 🤔
fixed